docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -105,18 +105,31 @@ When a Global Admin invites a user to join an account, the Global Admin selects
   - Has full administrative permissions across the account, including users and usage
   - Has Namespace Admin [permissions](#namespace-level-permissions) on all [Namespaces](/namespaces) in the account
 - **Developer**
-  - Can create and update Namespaces; has full control over [Workflows](/workflows)
-  - Has Namespace Admin permissions for each Namespace created by that user
-- **Read-Only:** Can only read information
-
-In addition, there are two roles that the Global Admin cannot assign:
+  - Can create Namespaces and modify Namespaces they create
+  - Has Namespace Admin [permissions](#namespace-level-permissions) on all [Namespaces](/namespaces) they've created
+    - Has read-level access to the list of all users within the account so they can add and remove users from a Namespace they've created
+    - Can assign Namespace-level permissions (either read-only or write) to users for a Namespace they've created
+  - Has write permissions for all Namespaces they create
+    - Can control [Workflows](/workflows), [Schedules](/workflows#schedule), etc within those Namespaces
+- **Read-Only (non-administrative)**
```

Comment:
In general I feel like there's not much to say about the Read-Only account role.

It basically is the "default" role that gives you access to the account.

I believe it allows you to view your own user profile page.

It does not give you any access to namespaces or even list users in the account (in the UI)